test/test_prototype_builtin_datasets.py

```diff
@@ -35,6 +35,7 @@ def test_coverage():
         )
 
 
+@pytest.mark.filterwarnings("error")
```

Comment:
On the other hand, playing devil's advocate here, right now we don't have a good way to detect warnings and fix them. I'm not sure anyone is monitoring them either. So I'll be OK to test this, see how often it produces actionable valid breakage and how often it drives us nuts.

Reply:
I'll look into that. Still, we need to at least include `torchdata`, since otherwise the first two warnings would have been allowed although they are clearly for us to act on.